toolkit/templates/forms/list-entry.html

```diff
@@ -13,7 +13,7 @@
 {% elif message %}
   <div class="message-wrapper">
 {% endif %}
-  <fieldset class="question {% if first_question %}first-question{% endif %}" id="{{ id }}">
+  <fieldset class="question {% if first_question %}first-question{% endif %}{% if hidden %} js-hidden related-information{% endif %}" id="{{ id }}">
```

Comment:
This is a thing we've done in a few places where we add extra spaces to classnames. 
If you remove the space after `class="question ` and put it before `first-question`, it's more correct.